docs/asciidoc/modules/ROOT/pages/overview/apoc.nodes/apoc.nodes.cycles.adoc

```diff
@@ -0,0 +1,39 @@
+////
+This file is generated by DocsTest, so don't change it!
+////
+
+= apoc.nodes.cycles
```

Comment:
Could we add a brief example to the docs?

Reply:
Added few examples in `usage/apoc.nodes.cycles.adoc`